src/AuditTrailModule/javasource/test_crm/tests/ExpectedLog.java

```diff
@@ -0,0 +1,58 @@
+package test_crm.tests;
+
+import static org.junit.Assert.*;
```

Comment:
If you are using a static import than its better import the necessary method instead of importing all.

Reply:
Okie, done.